frontend/src/components/form/DatePicker.vue

```diff
@@ -0,0 +1,68 @@
+<!--
+Displays a field as a date picker (can be used with v-model)
+-->
+
+<template>
+  <v-menu
+    ref="menu"
+    v-model="showDatePicker"
+    :close-on-content-click="false"
+    transition="scale-transition"
+    offset-y
+    min-width="290px">
+    <template v-slot:activator="{on}">
+      <v-text-field
+        v-model="date"
+        v-bind="$attrs"
+        prepend-icon="mdi-calendar"
+        readonly
+        v-on="on">
+        <!-- passing the append slot through -->
+        <template v-slot:append>
+          <slot name="append" />
+        </template>
+      </v-text-field>
+    </template>
+    <v-date-picker v-model="date" no-title scrollable>
```

Comment:
```suggestion
    <v-date-picker v-model="date" no-title first-day-of-week="1" scrollable>
```
Wochenstart am Montag

Reply:
Done